.github/workflows/push.yml

```diff
@@ -29,16 +29,29 @@ jobs:
     container:
       image: ghcr.io/klepsydra-technologies/kpsrbase:1.0.0
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           submodules: recursive
           fetch-depth: 0
 
+      - name: Build SDK
+        run: |
+          git clone https://github.com/klepsydra-technologies/kpsr-sdk.git
+          cd kpsr-sdk/
+          git switch KCE-84-Create-kpsr-sdk
+          git submodule update --init
+          mkdir build && cd build/
+          cmake ..
+          make -j$(nproc)
+          make install
+          rm -rf ${GITHUB_WORKSPACE}/kpsr-sdk/
+
       - name: Build
         run: |
           git config --global --add safe.directory ${GITHUB_WORKSPACE}
           mkdir build && cd build
-          cmake -DKPSR_WITH_DOXYGEN=true -DKPSR_WITH_DDS=false -DKPSR_WITH_ZMQ=false -DKPSR_TEST_PERFORMANCE=false -DKPSR_WITH_SOCKET=true -DKPSR_WITH_CODE_METRICS=true -DCMAKE_PREFIX_PATH=/opt/klepsydra/thirdparties -DCMAKE_BUILD_TYPE=Debug ../
+          cmake -DKPSR_WITH_DOXYGEN=true -DKPSR_WITH_ZMQ=false -DKPSR_TEST_PERFORMANCE=false -DKPSR_WITH_SOCKET=true -DKPSR_COMPILE_SPDLOG=false \
```

Comment:
Default `-DKPSR_COMPILE_SPDLOG=true` makes the CI fail, includes not finding "spdlog/spdlog.h", why?
Local build ok for both values.